test/src/main/resources/log4j2-test.properties

```diff
@@ -79,7 +79,7 @@ logger.16.name = org.apache.accumulo.server.util.ReplicationTableUtil
 logger.16.level = trace
 
 logger.17.name = org.apache.accumulo.core.clientImpl.ThriftScanner
-logger.17.level = info
+logger.17.level = debug
```

Comment:
Do we want to keep this change?

Reply:
For now, it's helping me debug some tests. In the long run, no.